get_ci_vm/setup.sh

```diff
@@ -30,9 +31,7 @@ apk upgrade
 apk add --no-cache "${PKGS[@]}"
 rm -rf /var/cache/apk/*
 
-pip3 install --upgrade pip
-pip3 install --no-cache-dir awscli
```

Comment:
This is the other risky change. I have no way of knowing if the "aws" from apk will fit the same needs as the pip one

We can no longer use pip, because the build breaks. "This environment is externally managed", and a long paragraph about python and PEP and whatnot.

Reply:
I can test this manually by pulling down the PR version of the image and re-tagging it "latest"...

Reply:
...yep, works just fine.

```bash
➜ podman pull quay.io/libpod/get_ci_vm:c20240125t184057z-f39f38d13
Trying to pull quay.io/libpod/get_ci_vm:c20240125t184057z-f39f38d13...
Getting image source signatures
...snip...
Writing manifest to image destination
b0b28b891d0e87b18cca461197a238a6d764eece9a3757f592f364720ef011ae

➜ podman tag quay.io/libpod/get_ci_vm:c20240125t184057z-f39f38d13 quay.io/libpod/get_ci_vm:latest

➜ hack/get_ci_vm.sh --list

# Initializing get_ci_vm

# Obtaining task listing from repository .cirrus.yml
...snip...normal...listing...
Build for debian-13
Build for fedora-38
Build for fedora-39
Build for fedora-39-aarch64
...snip...normal...listing...

➜ hack/get_ci_vm.sh Build for fedora-39-aarch64

# Initializing get_ci_vm

# Obtaining task listing from repository .cirrus.yml

# Obtaining instance type & image needed by 'Build for fedora-39-aarch64'

# Confirming and/or configuring AWS access.

# Will attempt to create a 'ec2vm' instance named 'cevich-fedora-podman-aws-arm64-c20240102t212217z-f39f38d13'
Note: Re-invoking this script will _not_ create additional instances.
Nor will it re-initialize any previously existing.  You will be
given an option to cleanup the instance upon exit.

...snip...normal...stuff...

# Accessing instance cevich-fedora-podman-aws-arm64-c20240102t212217z-f39f38d13: ssh -i /root/.ssh/ec2_containers -o StrictHostKeyChecking=no -o CheckHostIP=no -o UserKnownHostsFile=/root/.ssh/ec2_known_hosts -o ConnectTimeout=13 -q -q -t -t root@ec2-54-234-14-236.compute-1.amazonaws.com --
+ Entering Cirrus-CI environment
+ Changing into /var/tmp/go/src/github.com/containers/podman
+ Dropping into a bash login shell inside Cirrus-CI 'Build for fedora-39-aarch64' task environment
[root@ip-172-31-29-199 podman]#
logout

# Finalizing get_ci_vm
Offering to delete cevich-fedora-podman-aws-arm64-c20240102t212217z-f39f38d13 (might take a minute)
Note: It's perfectly safe to answer 'N'.  Simply re-run script again
with the same task argument, and you can access and/or delete the VM.

Are you sure you want to remove cevich-fedora-podman-aws-arm64-c20240102t212217z-f39f38d13 (y/N): y
Deleting cevich-fedora-podman-aws-arm64-c20240102t212217z-f39f38d13, for GCP this may take a moment or two.
```